src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -102,7 +102,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
     <AutomaticallyUseReferenceAssemblyPackages>true</AutomaticallyUseReferenceAssemblyPackages>
     <MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>1.0.0</MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>
-    
+    <CopyConflictingTransitiveContent>true</CopyConflictingTransitiveContent>
```

Comment:
I think this needs to be switched to false, per https://github.com/microsoft/msbuild/pull/4931#discussion_r415856633